### DISCUSSION TITLE
Default switch state

### DISCUSSION THREAD

**Discussion by BilalArif-34** [venturedrake/laravel-crm#60] [Q&A]
Hello, I am just a newbie in programming maybe my question is very simple. First of all thank you so much for keeping in open source and definitely it helps new comers to get an idea and a bit flavour of programming. I just want to ask when we add a new contact person in crm and the contact information there is a switch called primary which is by default off how can I turn the switch to ON state by default. Here is the blade code in phone-edit.blade.php. I would appreciate your help.

`<div class="form-group" wire:ignore>
                    <label>{{ ucfirst(__('laravel-crm::lang.primary')) }}</label>
                    <input type="checkbox" wire:model="primary.{{ $value }}" name="phones[{{ $value }}][primary]" data-toggle="toggle" data-size="sm" data-on="Yes" data-off="No" data-onstyle="success" data-offstyle="danger">
                    @error('primary.'.$value) <span class="text-danger invalid-feedback-custom">{{ $message }}</span>@enderror
                </div>`

**Comment by andrewdrake** [comment]
Hi @BilalArif-34,

Thanks for the thank your :) Sorry I somehow missing this question. 

This is actually a good idea, I will add a setting which can use the default the primary toggle to true. 

I have a few updates coming, will release this at the same time.
